crates/external/rpc/src/traits.rs

```diff
@@ -6,13 +6,17 @@ use alloy_primitives::Log;
 use alloy_sol_types::SolEvent;
 use async_trait::async_trait;
 use core::time::Duration;
-use jsonrpsee::core::ClientError;
+use alloy_json_rpc::{Response, ResponsePayload};
+use jsonrpsee::{core::ClientError, types::ErrorObjectOwned};
+use jsonrpsee::core::RpcResult;
 use kona_interop::{CROSS_L2_INBOX_ADDRESS, ExecutingMessage, SafetyLevel};
 use tokio::time::error::Elapsed;
 
 /// Failures occurring during validation of [`ExecutingMessage`]s.
 #[derive(thiserror::Error, Debug)]
 pub enum ExecutingMessageValidatorError {
+    /// Safety level not sufficient
+    Unsafe,
```

Comment:
nice! would be nice if variant wrapped the expected `SafetyLevel` and the safety level that we got, but maybe not possible with current op-supervisor code, so then it's out of scope for now

Reply:
I made manual parsing, but it looks ugly, maybe it's better to use regexp x)